# Scripts description 


We made following the ipyrad default pipeline modifying only the next parameters:


1. clustering treeshold: 0,97 and 0,95 
1. min_samples_locus: 1, chosen so as not to discard the sequences from possible exclusive symbionts of a particular sample
1. max_alleles_consens: 4 to prevent the influence of allelic differences for identifying a differences between microbial sequences.


For each strategy, the .loci file generated by ipyrad, that presents the reads associated with each locus, was saved with the name of the individual from which the sample originated. The R script written by [Holmes and Rabosky, 2018](https://peerj.com/articles/4662/) was used to take the first sequence of each locus and group them into a fasta file that was later processed using the [blastn tool] (https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastDocs&DOC_TYPE=Download
) from the National Center for Biotechnology Information (NCBI).
