# Seeking microbiome sequences within RADSeq data


In this project we are scanning microbiological diversity within RADSeq data from bark beetles of the genus Dendroctonus to retrieve metagenomic information from microbial communities associated with bark beetles.

The RADSeq data we use comes from the publication of [Godefroid et al, 2019](https://www.sciencedirect.com/science/article/abs/pii/S1055790319302441), that was generated to build a phylogeny of the bark beetle Dendroctonus (17 out of 20 currently known species) [NCBI Bioproject: PRJNA530572](https://www.ncbi.nlm.nih.gov/bioproject/?term=txid77165[Organism:noexp]).

Scrpits described bellow are available in [bin](https://github.com/daniel-acga/Metagenomics-RAD-Insect-/blob/master/bin/Holmes%202018%20R%20blast), associated data are in [data](link) and results are in [output](link).



## Methods

We analyz the data using the iPyrad v.0.7 pipeline .28 [Eaton & Overcast, 2020](https://academic.oup.com/bioinformatics/article-abstract/36/8/2592/5697088).

### Quality Filtering
Qality filtering and reads assignment were performed using the default parameters of ipyrad. Reads were assembled using two clustering threshold values (0.97 and 0.95) that correspond to similarity between reads.



### Gene assembly
We tried three approaches to group the shared loci between the samples analyzed:

1. de novo assembly
1. using a fungi reference genome
1. using an insect reference genome as filter

The first type of assembly (de novo) generates clusters by using a global alignment grouping algorithm, using the USEARCH program [Edgar, 2010] (https://www.osti.gov/biblio/1137186), which allows the incorporation of indelible variations when identifying homology. 

The second takes as a reference a genome chosen by the user to identify homologies and discards everything that does not resemble it. To generate the assembly we used the complete genome sequence of one of the characteristic symbiote fungi of this bark beetle genus, Grosmannia clavigera [NCBI BioProject: PRJNA39837 DiGuistini et al, 2011](https://www.pnas.org/content/108/6/2504.short).

The third one (de novo - reference) removes any reading that matches the reference sequence and retains only mismatched reads that uses in a de novo assembly (Eaton & Overcast, 2020). This last approach was used to identify new loci in the samples previously labeled by the original article as Dendroctonus ponderaseae, and also agaonst the universe of samples as another assemble, using as a reference to the sequence of the complete genome project Dendroctonus ponderosae [NCBI BioProjectI: PRJNA162621, Keeling et al, 2013](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-3-r27).

For each strategy, the .loci file generated by ipyrad, which presents the reads associated with each locus, was saved with the name of the individual from which the sample originated. The R script written by [Holmes and Rabosky, 2018](https://peerj.com/articles/4662/) was used to take the first sequence of each locus and group them into a fasta file that was later processed using the [blastn tool] (https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastDocs&DOC_TYPE=Download
) from the National Center for Biotechnology Information (NCBI).

ADD BLAST METODOLOGY

blastn -task dc-megablast -db nt -query [your header].fa -outfmt "6 qseqid sseqid staxids sskingdoms pident mismatch" -out [your header]_out.txt -num_threads 2  -max_target_seqs 20

## Results
